### ISSUE TITLE
Update to phpcs 3.3.0

### ISSUE THREAD

**Issue by tomasfejfar** [consistence/coding-standard#40]
This would allow to drop "blank line after class opening bracket" and "blank line before class closing bracket" rules in custom stylesheets [thanks to new options](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizwhitespacefunctionspacing).

**Comment by VasekPurchart** [comment]
Of course I will update to it, as I always do, but only after it is released :)

https://github.com/squizlabs/PHP_CodeSniffer/milestone/7

**Comment by tomasfejfar** [comment]
Ouch, yesterday wasn't my strongest day 🙄 I made two dumb mistakes in one evening. Sorry for inconvenience.

**Comment by VasekPurchart** [comment]
No problem, it happens to all of us :)

**Comment by VasekPurchart** [comment]
PHPCS 3.3 is out, but since there were quite a few issues, the focus was on them and I didn't get to use the new options yet, but I keep them in mind :)

**Comment by VasekPurchart** [comment]
Ok, new options not needed for Consistence ruleset, all cleared up :)
